### ISSUE TITLE
basic icons not found or missing

### ISSUE THREAD

**Issue by Ash3781** [yeyushengfan258/Lyra-icon-theme#4]
hey, first of all i am a huge fan of your icon sets, tho if i apply the icon theme the most basic files like .txt and .sh files do not have icons, same goes for .xz, .gz and .html files, is there a mistake on my end or are the icons just missing for that?
![Screenshot_20](https://user-images.githubusercontent.com/37660156/154307717-bbf4efbc-869c-4734-a28b-ec0e7795c5e5.png)

**Comment by yeyushengfan258** [comment]
There are icons, and I can see and open .txt files here.

**Comment by Ash3781** [comment]
well, opening and closing isn't a issue, the only issue is that the files like .txt, .sh etc got such a different style than the folders that it does not fit, is there something else interfering or is there some mistake on my end?

**Comment by Ash3781** [comment]
but then again if i try a different icon pack like beautyline all icons are changed, that's what's confusing about it
[
![2022-02-20_12-34 cleaned](https://user-images.githubusercontent.com/37660156/154840539-97e40597-7439-43cc-8050-6f330ca841ec.png)
](url)

**Comment by Ash3781** [comment]
i did a little bit of research about icon packs, and i noticed that there is no file for the .txt etc file there, do i have to download them separately somewhere else?

**Comment by yeyushengfan258** [comment]
You are not wrong, these files like .txt are not finished yet. They will be uploaded in future updates.

**Comment by Ash3781** [comment]
i've never seen how a icon pac gets updated, do i just have to re-download it in like a week / month etc or does it actually tell me there is a update?

**Comment by yeyushengfan258** [comment]
There is no prompt, but if there is an update, you go to the store to find this icon and it will show the update.
You can wait for a while, the next update will only update a part because of the large number, when you remember, you can check whether it is updated.

**Comment by yeyushengfan258** [comment]
I will focus on improving .txt and other types of icons for this icon.

**Comment by Ash3781** [comment]
Are there still updates planned, or is this repository abandoned?
